apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -321,7 +321,7 @@ loginView { state, setState } = React.fragment
                 log "Fetching user succeeded"
                 setState $ setLoggedInUser $ Just user
           , launchAff_:
-              Aff.runAff_ (either (setState <<< setAlert <<< errorAlert) pure)
+              Aff.runAff_ (setState <<< setAlert <<< either errorAlert (const Nothing))
```

Comment:
Here I've added @vapaj's suggestion to remove the error message once the aff succeeds at anything, this isn't the perfect handling, but it handles most of the use cases and is very straightforward.